src/lib.rs

```diff
@@ -20,7 +20,24 @@ fn default_result<T>() -> Result<T, Error> {
     ))
 }
 
-/// Return the MTU of the interface that is used to reach the given remote socket address.
+/// Return the maximum transmission unit (MTU) of the local network interface towards the
+/// destination `SocketAddr` given in `remote`.
+///
+/// The returned MTU may exceed the maximum IP packet size of 65,535 bytes on some
+/// platforms for some remote destinations. (For example, loopback destinations on
+/// Windows.)
+///
+/// # Examples
+///
+/// ```
+/// use std::net::ToSocketAddrs;
+///
+/// use mtu::get_interface_mtu;
+///
+/// let saddr = "localhost:443".to_socket_addrs().unwrap().next().unwrap();
+/// let mtu = get_interface_mtu(&saddr).unwrap();
+/// println!("MTU towards {:?} is {}", saddr, mtu);
+/// ```
```

Comment:
```suggestion
/// ```
/// # use std::net::ToSocketAddrs;
/// # use mtu::get_interface_mtu;
/// let saddr = "127.0.0.1:443".parse().unwrap();
/// let mtu = get_interface_mtu(&saddr).unwrap();
/// println!("MTU towards {:?} is {}", saddr, mtu);
/// ```
```

Just some nits